lib/read_parsers.cc

```diff
@@ -20,6 +20,21 @@ namespace khmer
 namespace read_parsers
 {
 
+void
+Read::write_to(std::ostream& output)
```

Comment:
I think `write_to` is a better name than `write_record`, since the invocation is something like `read.write_to(std::cout)`. Thoughts?